home/service/details.py

```diff
@@ -258,12 +261,12 @@ def _get_context(self):
 
         return {
             "entity": self.publication_dataset_metadata,
-            "entity_type": EntityTypes.PUBLICATION_DATASET.value,
+            "entity_type": PublicationDatasetEntityMapper.find_moj_data_type,
             "parent_entity": self.parent_entity,
             "platform_name": friendly_platform_name(
                 self.publication_dataset_metadata.platform.display_name
             ),
-            "parent_type": EntityTypes.PUBLICATION_COLLECTION.name.lower(),
+            "parent_type": DatabaseEntityMapper.datahub_type.lower(),
```

Comment:
I think we can drop the `lower()` here, and change templates accordingly. We're not surfacing datahub types to users, so the case shouldn't matter as long as we are consistent.

That said, if we do a bigger refactor, then I think this is an example where we can eliminate the Datahub type entirely and replace it with our own entity type. The end state of that work should be that the mapping from Datahub to Find MoJ data entity types is done entirely within the `lib/datahub-client` modules, and Find MoJ data does not reference datahub entity types at all (as @hjribeiro-moj suggested last week)